misc/python/materialize/visualize_pprof_profile/profile.proto

```diff
@@ -0,0 +1,233 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
As said in pprof, this is received from a CURL. Not sure if we need to do something to account for this apache license.

A potential worry is the `pprof.gz` provided doesn't match this proto, but given protobufs are supposed to be backwards compatible and because this `.proto` file (one used for pprof files) is most likely stable, I thought it'd be okay

Reply:
Noticing this is failing the check-copyright lint test. I can fix this via changing the header to 
```
// Copyright Materialize, Inc. and contributors. All rights reserved.
//
// Use of this software is governed by the Business Source License
// included in the LICENSE file.
//
// As of the Change Date specified in that file, in accordance with
// the Business Source License, use of this software will be governed
// by the Apache License, Version 2.0.
```
but this seems quite suspicious to just override a copyright header like that. Any advice?

Reply:
I have added an ignore for the copyright check in the linters.